okhttp-logging-interceptor/src/test/java/okhttp3/logging/LoggingEventListenerTest.java

```diff
@@ -147,7 +147,7 @@ public void secureGet() throws Exception {
         .assertLogMatch("secureConnectStart")
         .assertLogMatch("secureConnectEnd: Handshake\\{"
             + "tlsVersion=TLS_1_[23] "
-            + "cipherSuite=(?:TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384|TLS_AES_128_GCM_SHA256) "
+            + "cipherSuite=(?:TLS_(?:ECDHE_ECDSA_WITH_|)AES_(?:256_GCM_SHA384|128_GCM_SHA256)) "
```

Comment:
This reads very intentional on the cipher, we could instead just check it starts with TLS_?

Reply:
Makes sense, less likely to break in the future. I'll amend.